_includes/tutorials/aggregating-sum/kstreams/code/src/main/java/io/confluent/developer/AggregatingSum.java

```diff
@@ -64,37 +53,35 @@ public Topology buildTopology(Properties envProps,
         // Apply SUM aggregation
         .reduce(Integer::sum)
         // Write to stream specified by outputTopic
-        .toStream().to(outputTopic, Produced.with(Serdes.String(), Serdes.Integer()));
+        .toStream().mapValues(v -> v.toString()).to(outputTopic, Produced.with(Serdes.String(), Serdes.String()));
```

Comment:
one nit change to `(v -> v + " total sales")`

Reply:
Thank you for the review and comment @bbejeck I updated the code/tests/output with your suggestion in commit https://github.com/confluentinc/kafka-tutorials/pull/814/commits/8bbc18c43deb6632d79bcf246209cfcfffa37f0b